tfhe/docs/fundamentals/compress.md

```diff
@@ -41,6 +41,84 @@ fn main() {
 }
 ```
 
+## Ciphertexts compressed while encrypted
+
+Ciphertexts can also be compressed while encrypted.
+They can be encrypted after any number of homomorphic operations.
+
+The API uses a list which can compress many ciphertexts of different types.
+Using a single list allows a better compression ratio.
+
+Using multiple lists (for example to compress ciphertexts at different times) is also possible but will use more space.
+
+This example shows how to compress and decompress a list containing 4 messages (one 32 bits integer, one 64 bits integer, one boolean and one 2 bits integer):
+
+```rust
+use crate::prelude::*;
+use crate::shortint::parameters::glwe_compression::COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS;
+use crate::{
```

Comment:
crate -> tfhe, the doctests are failing on that I believe

Reply:
Done